packages/calcite-components/src/components/shell-panel/shell-panel.scss

```diff
@@ -146,70 +146,112 @@
   border-inline: 1px solid var(--calcite-shell-panel-border-color, var(--calcite-color-border-3));
 }
 
-:host([layout="horizontal"]) .container {
+:host([layout="horizontal"]) .container,
+:host([layout="horizontal"]) .content-container {
   @apply w-full h-auto;
   flex-direction: column;
 }
 
-:host(:hover),
-:host(:focus-within) {
-  .separator:not(:hover):not(:focus) {
-    @apply opacity-100;
-    background-color: var(--calcite-color-border-3);
-  }
+.resize-handle {
```

Comment:
This is causing a bit of overlap:
<img width="400" alt="Screenshot 2025-04-21 at 10 44 28 AM" src="https://github.com/user-attachments/assets/70b8e90b-671f-44a5-ac5a-5b77fba3d698" />
<img width="400" alt="Screenshot 2025-04-21 at 10 44 10 AM" src="https://github.com/user-attachments/assets/ae18cad5-94c2-4ed7-a1e1-802d39a3f6c0" />

Reply:
fixed!